modules/core/src/errors.ts

```diff
@@ -56,6 +56,13 @@ export class P2wshUnsupportedError extends BitGoJsError {
   }
 }
 
+export class P2trUnsupportedError extends BitGoJsError {
+  public constructor(message?: string) {
+    super(message || 'p2tr not supported by this coin');
+    Object.setPrototypeOf(this, P2trUnsupportedError.prototype);
```

Comment:
FYI we don't need these any longer

this was for `target: es5` times https://www.dannyguo.com/blog/how-to-fix-instanceof-not-working-for-custom-errors-in-typescript/#upgrade-the-compilation-target